<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransformWidget</class>
 <widget class="QDialog" name="TransformWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>454</width>
    <height>434</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Transform</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="activate">
     <property name="text">
      <string>Transform active</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0,0,10">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="1,3">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>100000</height>
          </size>
         </property>
         <property name="text">
          <string>num</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="num">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="1,3">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>100000</height>
          </size>
         </property>
         <property name="text">
          <string>vx</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="vx">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="1,3">
       <item>
        <widget class="QLabel" name="label_11">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>100000</height>
          </size>
         </property>
         <property name="text">
          <string>vy</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="vy">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_12" stretch="1,3">
       <item>
        <widget class="QLabel" name="label_12">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>100000</height>
          </size>
         </property>
         <property name="text">
          <string>vz</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="vz">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="Compute">
     <property name="text">
      <string>Compute</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_13">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="inputLabel">
         <property name="text">
          <string>Input</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="InputList"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="outputLabel">
         <property name="text">
          <string>Output</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="OutputList"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_14" stretch="10,0,0">
     <item>
      <spacer name="horizontalSpacer_10">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="Cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Apply">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>num</tabstop>
  <tabstop>vx</tabstop>
  <tabstop>vy</tabstop>
  <tabstop>vz</tabstop>
  <tabstop>Compute</tabstop>
  <tabstop>InputList</tabstop>
  <tabstop>OutputList</tabstop>
  <tabstop>Cancel</tabstop>
  <tabstop>Apply</tabstop>
  <tabstop>activate</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>Compute</sender>
   <signal>clicked()</signal>
   <receiver>TransformWidget</receiver>
   <slot>compute()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>380</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>431</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Cancel</sender>
   <signal>clicked()</signal>
   <receiver>TransformWidget</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>386</x>
     <y>426</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>399</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Apply</sender>
   <signal>clicked()</signal>
   <receiver>TransformWidget</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>446</x>
     <y>426</y>
    </hint>
    <hint type="destinationlabel">
     <x>365</x>
     <y>399</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>activate</sender>
   <signal>clicked(bool)</signal>
   <receiver>TransformWidget</receiver>
   <slot>activate(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>33</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel">
     <x>-11</x>
     <y>10</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>compute()</slot>
  <slot>activate(bool)</slot>
 </slots>
</ui>
